tests/cdi/golden-pvc.yml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: cdi-test
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: ${PVC_NAME}
+    annotations:
+      kubevirt.io/storage.import.endpoint: "${EP_URL}"
+      kubevirt.io/storage.import.secretName: "${EP_SECRET}"
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 10Gi
+    storageClassName: kubevirt
```

Comment:
I think you should not use the kubevirt storage class but allow it to be created in the default storage class.

Reply:
blocked by this one: https://github.com/kubevirt/kubevirt-ansible/issues/304

Reply:
submit a patch here https://github.com/kubevirt/kubevirt-ansible/pull/305, PTAL